src/rest/GroupByRequest.ts

```diff
@@ -90,6 +91,11 @@ export interface IGroupByRequest {
    */
   allowedValues?: string[];
 
+  /**
+   * The type of pattern used in {@link IGroupByRequest.allowedValues}. Possible values are legacy, wildcards and regex.
```

Comment:
Would it be good to mention the default value? Based on the next file and the unit test, it seems to be `Wildcard`?

Reply:
SearchApi side, the default value is actually `legacy`. We set it to `wildcards` only for facet search. I can add this to the docs.